doc/content/verification_and_validation/ver-1kb.md

```diff
@@ -10,7 +10,7 @@ Two enclosures are separated by a membrane that allows diffusion according to He
 
 This verification problem is taken from [!cite](ambrosek2008verification).
 
-This setup describes a diffusion system in which tritium T$_2$ is modeled across a one-dimensional domain split into two enclosures. The total length of this domain is defined by the number of segments (20 segments) and the node size of $1.25 \times 10^{-5}$ m, yielding a total system length of $2.5 \times 10^{-4}$ m. The system operates at a constant temperature of 500 Kelvin. Initial tritium pressures are specified as $10^{5}$ Pa for Enclosure 1 and $10^{-10}$ Pa for Enclosure 2.
+This setup describes a diffusion system in which tritium T$_2$ is modeled across a one-dimensional domain split into two enclosures. The total length of this domain is defined by the number of segments (100 segments) and the node size of $1.25 \times 10^{-5}$ m, yielding a total system length of $2.5 \times 10^{-4}$ m. The system operates at a constant temperature of 500 Kelvin. Initial tritium pressures are specified as $10^{5}$ Pa for Enclosure 1 and $10^{-10}$ Pa for Enclosure 2.
```

Comment:
You actually did not update the number of segments in the input file. 

Also, you should update the node size being documented here.